src/parachain/issue.ts

```diff
@@ -286,9 +290,14 @@ export class DefaultIssueAPI extends DefaultTransactionAPI implements IssueAPI {
         }
     }
 
-    async execute(requestId: string, btcTxId?: string, merkleProof?: Bytes, rawTx?: Bytes): Promise<void> {
+    async execute(requestId: string, btcTxId?: string, merkleProof?: Bytes, rawTx?: Bytes, electrsAPI?: ElectrsAPI): Promise<void> {
```

Comment:
If I understand this correctly, it would not be possible to do `issue.execute("requestId")` directly even though the other parameters are optional. IMO that is not a good change since it's confusing that optional parameters are required for the function to complete successfully and it requires an extra step for whoever is consuming this API.

What's the reason for removing electrs here? I'd prefer to keep this in this function so that the execute can be done as simple as possible.